docs/content/providers/overview.md

```diff
@@ -35,9 +35,10 @@ Below is the list of the currently supported providers in Traefik.
 | [Rancher](./rancher.md)               | Orchestrator | Label                      |
 | [File](./file.md)                     | Manual       | TOML/YAML format           |
 | [Consul](./consul.md)                 | KV           | KV                         |
-| [etcd](./etcd.md)                     | KV           | KV                         |
+| [Etcd](./etcd.md)                     | KV           | KV                         |
```

Comment:
while `etcd` has no RFC, it is commonly referred to as `etcd` by the publishers of the project. You can also find examples all over the web where the `e` is capitalized, however, I noticed we use `Etcd` consistently throughout the documentation so this makes sense.

Reply:
Also when used as a proper noun, usually capitalized.